internal_transfers/actions/src/utils.ts

```diff
@@ -22,7 +22,12 @@ export async function getTxDataFromHash(
   const { from, hash, blockNumber, logs: readonlyLogs } = transaction;
   const logs = readonlyLogs.map((log) => {
     return {
-      ...log,
+      // Prune TxReceipt:
+      // could use `...log` here, but that comes with undesired bulk
+      // e.g. https://jsonblob.com/1106016587013701632
+      index: log.index,
+      address: log.address,
+      data: log.data,
       // had to Map it to make a copy of a readonly field.
       topics: log.topics.map((value) => value),
```

Comment:
Another optional shortcut:

```suggestion
      topics: [...log.topics],
```